### ISSUE TITLE
Allow to change the entity domain

### ISSUE THREAD

**Issue by gianklug** [tuya/tuya-home-assistant#221]
If someone uses a Tuya Switch for a Lamp, we should be able to change the Homeassistant Entity domain from `switch.xxxxxxxxxxx` to `light.xxxxxxxxxx`.

This would have the following advantages:
* Automatic Lightbulb icon instead of Flash
* Easier handling inside of lighting groups
* When using a brige for e.g. homekit, the Device type would be Light there too

This was already possible using the inofficial localtuya extension. It would be nice if this extension would support it too.

**Comment by johnjoemorgan** [comment]
For any integration/addon you can change the domain to light using light.yaml

  - platform: switch
    name: Flood Lights
    entity_id: switch.tyexxxxxxxxxxxxxxb8q

That will give you a light entity for the Tuya switch

**Comment by gianklug** [comment]
@johnjoemorgan but then I will have duplicate entities which is kinda annoying

**Comment by johnjoemorgan** [comment]
@gianklug Yes - again agreed.  I disable the switch.tyXXX when I create the new light.tyXXX.  Not elegant but it works
